app/assets/stylesheets/administrate/base/_typography.scss

```diff
@@ -23,7 +23,7 @@ p {
 
 a {
   color: $action-color;
-  text-decoration: none;
+  text-decoration-skip: ink;
```

Comment:
Hound should recognize this going forward from https://github.com/thoughtbot/administrate/commit/6e88449ff412189504a60d813e9f24ecc49f0dae#diff-b9ee5b6eb41f94a0d74486255de4ad10R128